cluster-operator/src/main/java/io/strimzi/operator/cluster/model/metrics/StrimziMetricsReporterModel.java

```diff
@@ -0,0 +1,96 @@
+/*
+ * Copyright Strimzi authors.
+ * License: Apache License 2.0 (see the file LICENSE or http://apache.org/licenses/LICENSE-2.0.html).
+ */
+package io.strimzi.operator.cluster.model.metrics;
+
+import io.strimzi.api.kafka.model.common.HasConfigurableMetrics;
+import io.strimzi.api.kafka.model.common.metrics.StrimziReporterMetrics;
+import io.strimzi.kafka.oauth.common.ConfigException;
+import io.strimzi.operator.common.model.InvalidResourceException;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Optional;
+import java.util.regex.Pattern;
+import java.util.regex.PatternSyntaxException;
+
+/**
+ * Represents a model for components with configurable metrics using Strimzi Reporter
+ */
+public class StrimziMetricsReporterModel {
+
+    /**
+     * Name of the Strimzi metrics port
+     */
+    public static final String METRICS_PORT_NAME = "http-prometheus";
```

Comment:
```suggestion
    public static final String METRICS_PORT_NAME = "tcp-prometheus";
```

Reply:
That was a wrong suggestion from me. Sorry.

Reply:
Refactored